app/assets/javascripts/success_prospects.es6

```diff
@@ -0,0 +1,17 @@
+$(function() {
+  $('#success-prospect-text-area').hide();
+  var option_name = $("input[type=radio][name='merits_assessment[success_prospect]']:checked").val()
+  if (typeof(option_name) != "undefined" && option_name != 'better') {
+    displayTextArea(option_name)
+  }
+
+  $('.success_prospect_option').click(function() {
+    displayTextArea($(this).val());
+  });
+});
+
+function displayTextArea(option_name) {
```

Comment:
The behaviour of this function is specific to `success-prospect` where as the name implies it can be used on any text area. I'd suggest the function name reflect the tight scope of the function: 
```javascript
function showSuccessProspectTextArea(option_name) {
```

Reply:
changed